### PR TITLE
TAJO-1534: DelimitedTextFile return null instead of a NullDatum 

### PR DESCRIPTION
**Pull request by jinossy**

### CODE REVIEW

File: tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/rcfile/RCFile.java [hunk 1]
```diff
@@ -1255,16 +1255,21 @@ public void init() throws IOException {
...
           skippedColIDs[tid] = false;
 
-          SelectedColumn col = new SelectedColumn();
           col.colIndex = tid;
           col.runLength = 0;
           col.prvLength = -1;
           col.rowReadIndex = 0;
           selectedColumns[i] = col;
           colValLenBufferReadIn[i] = new NonSyncDataInputBuffer();
+        }  else {
```
Comment (blrunner) [inline tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/rcfile/RCFile.java]: The newly added code is the mostly same as those of existing codes.

Comment (jinossy) [inline tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/rcfile/RCFile.java]: Thank you for nice finding.
I've remove the duplicated codes

**Comment by blrunner** [comment]
+1

It looks good to me and all unit test cases finished successfully.
And if possible, could you add 0.11.0 to the fix version of JIRA issue?

**Comment by jinossy** [comment]
@blrunner 
Thank you for your review. I will commit soon

**Pull request closed without merging**